Makefile

```diff
@@ -72,8 +75,14 @@ save: save-genesis save-join
 
 genesis: build-genesis
 
+ifeq ($(PREFETCH_IMAGES), true)
 build-genesis: Dockerfile.genesis cni.tgz env.sh helm genesis-images.tar kubelet kubelet.service.template
```

Comment:
It might be a bit cleaner to build the dependencies dynamically so that you don't have to repeat the body of the target:

```makefile
GENESIS_BUILD_DEPS := Dockerfile.genesis cni.tgz env.sh helm kubelet kubelet.service.template

ifeq ($(PREFETCH_IMAGES), true)
GENESIS_BUILD_DEPS += genesis-images.tar
endif
```